File: rolemadelen/programmers/120893_대문자와_소문자.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string my_string) {
    string answer = "";
    
    for(char ch : my_string) {
        if(ch >= 'a' && ch <= 'z') answer += (ch - 32);
        else answer += (ch + 32);
    }
    
    return answer;
}

File: simsu/1주차/120893_대문자와_소문자.js
function solution(my_string) {
    let answer = '';
    for(let i in my_string) {
        let ascii_num = my_string.charCodeAt(i)
        if(ascii_num<=90) answer += String.fromCharCode(ascii_num + 32);
        else answer += String.fromCharCode(ascii_num - 32);
    }
    return answer;
}